src/yuzu/main.cpp

```diff
@@ -1632,7 +1633,8 @@ void GMainWindow::BootGame(const QString& filename, u64 program_id, std::size_t
     Settings::LogSettings();
 
     if (UISettings::values.select_user_on_boot) {
-        SelectAndSetCurrentUser();
+        if (SelectAndSetCurrentUser() == false)
+            return;
```

Comment:
```suggestion
        if (!SelectAndSetCurrentUser()) {
            return;
        }
```
We always use braces around if statements.

Reply:
Maybe do the same with `loadROM` since you are already touching this part of the code.